backend/muster-roll-service/Muster-Roll-Service-1.0.0.yaml

```diff
@@ -154,6 +156,9 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/IndividualEntry'
+        additionalDetails:
+          type: object
+          description: This can be used to store skillset that is a requirement for Mukta. This is an updatable field from the user's input.
```

Comment:
The skillset related description only applies for the additionalDetails property of AttendanceEntry. So remove the first line of the description.

Reply:
Removed the first line of the description